### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_DeployDacpac_DeployDacpacSummaryScreen_ScreenReader: Voiceover does not announce the details in the table. 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6822]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #A11y_ADS_Schema_Dacpac_Backup; #Azure Data Studio; #DesktopApp; #Win10; #MAS1.3.1; #VoiceOver;

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: e8ccc65
Electron: 4.2.7
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
MAC 10.14.6
Tools Name: Voiceover
MAS References: MAS1.3.1

**Repro Steps:**
1.Launch Azure Data Studio application.
2.Connect to server with windows credentials.
3.Navigate to Extensions from left menu.
4.Search & install Dacpac extension.
5.Navigate to Connections from left menu.
6.Expand the server connected in step 2.
7.Right click an on prem database and select Data-tier application wizard from Context menu.
8.Select first radio button.
9.Navigate till next button & hit enter.
10.Provide all the details for Deploy Dacpac & hit next.
11.Navigate till Next button & hit enter.
12.Start screen reader as Jaws.
13.Navigate to the header row using  control+option+arrow key

**Actual:**
**Issue1:**
On navigating to the table using  control+option+arrow key, the table is announced but the details inside the table are not. They are not accessible by control+option+arrow key.

**Issue2:**
On selecting the rows of the table using mouse the details are read but the header value in each cell is announced twice.

**Same issue is observed in the following pages:**
1.Extract Dacpac Summary Screen
Follow the exact same steps mentioned above but select the second radio button.
2.Import Bacpac Summary Screen
Follow the exact same steps mentioned above but select the third radio button.

**Expected:**
**Issue1:**
On navigating to the cells of the table using control+option+arrow key, the details need to be announced.

**Issue2:**
The header value in each cell should be announced only once.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: [https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
The screen reader user will not receive the information of the details present inside the table.

MAS Reference:
MAS 1.3.1-
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

**Attachment for Reference:**
[#6822A11y_AzureDataStudio(Schema,Dacpac,Backup)_DeployDacpac_DeployDacpacSummaryScreen_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3519706/6822A11y_AzureDataStudio.Schema.Dacpac.Backup._DeployDacpac_DeployDacpacSummaryScreen_ScreenReader.pptx)

**Comment by fsteffi** [comment]
On navigating to the table using **control+option+arrow key,**  the table as a whole is announced but the details inside the table , the cells are not. The details are read in the video because they are selected using  tab key which is incorrect since they are not actionable and tab movement should not be there in the table.

**Comment by AbhiramDeshmukh** [comment]
#A11yMAS;#A11yTCS

**Comment by Benjin** [comment]
Tested on macOS with the latest insider build, and I'm able to navigate to the table and around the cells using Ctrl + Opt + Arrows, getting Voiceover narrations of the cell contents the entire way.  There's extra being narrated ("empty group" is said after each value), but the fundamental scenario works.  Also, tab movement was removed from the table due to another change by @udeeshagautam.
